src/_common/content/components/media-item/media-item.ts

```diff
@@ -130,6 +132,12 @@ export default class AppContentMediaItem extends Vue {
 		return !this.imageLoaded && !GJ_IS_SSR;
 	}
 
+	get useResponsiveImg() {
+		// For animated media items (gifs):
+		// Always return the direct url because gifs do not get returned by mediaserver.
+		return !!this.mediaItem && !this.mediaItem.is_animated && !!this.mediaItem.mediaserver_url;
```

Comment:
Can't you just do `return !!(this.mediaItem?.is_animated && this.mediaItem?.mediaserver_url)` now?